src/index.html

```diff
@@ -11,7 +11,151 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
```

Comment:
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
      >
```
too long for one line